guided-match/ddl.sql

```diff
@@ -0,0 +1,204 @@
+CREATE INDEX COAG_IDX4 on COMMERCIAL_AGREEMENTS  (authorisation_email);
+*/
+
+/* Neo4J 
+CREATE TABLE domain_question_outcomes (
+   outcome_id                       INTEGER PRIMARY KEY,
+   journey_question_id              INTEGER NOT NULL,
+   outcome_evaluator_code           VARCHAR(10) NOT NULL, 
+   journey_answer_id                INTEGER NOT NULL,
+   modifier_journey_name            VARCHAR(10) NOT NULL
+);
+*/
+/* Neo4J
+CREATE TABLE explicit_question_outcomes (
+   outcome_id                       INTEGER PRIMARY KEY,
+   journey_question_id              INTEGER NOT NULL,
+   outcome_evaluator_code           VARCHAR(10) NOT NULL, 
+   journey_answer_id                INTEGER NOT NULL,
+   resulting_journey_question_id    INTEGER NOT NULL
+);
+*/
+
+/* Neo4J
+CREATE TABLE journey_answers ( --Need to check this one out with Dave H as attributes on model are different to descriptions on confluence page
+  journey_answer_id                INTEGER PRIMARY KEY,
+  journey_question_id              INTEGER NOT NULL,
+  answer_type                      VARCHAR(50),
+  answer_label                     VARCHAR(100),
+  default_ind                      CHAR(1), -- Y or N default value will be N as only one can be Y
+  valid_value                      VARCHAR(200),
+  url                              VARCHAR(200), -- notsure required anymore
+  regex_validation                 VARCHAR(2000)  -- notsure required anymore
+);
+*/
+
+CREATE TABLE journey_instance_questions (
+  journey_instance_question_id      INTEGER PRIMARY KEY,
+  journey_question_id               INTEGER NOT NULL,
+  question_order                    INTEGER NOT NULL,
+  question_text                     VARCHAR(2000) NOT NULL,
+  answer_id                         INTEGER NOT NULL,
+  answer_text                       VARCHAR(2000) 
+);
+CREATE INDEX JOIQ_IDX1 on JOURNEY_INSTANCE_QUESTIONS (journey_questiuon_id);
+CREATE INDEX JOIQ_IDX2 on JOURNEY_INSTANCE_QUESTIONS (answer_id);
+
+
+CREATE TABLE journey_instances (
+  journey_instance_id               INTEGER PRIMARY KEY,
+  journey_start_date                DATE NOT NULL,
+  journey_end_date                  DATE NULL
+);
+CREATE INDEX JOIN_IDX1 ON JOURNEY_INSTANCES(journey_start_date);
+
+/* Neo4J
+CREATE TABLE journey_questions (
+  journey_question_id               INTEGER PRIMARY KEY,
+  question_id                       INTEGER NOT NULL,
+  question_order                    SMALLINT NOT NULL ,
+  regex_validation                  VARCHAR(2000),
+  min_threshold_range               NUMBER(12,2),
+  max_threshold_range               NUMBER(12,2),
+  threshold_value                   NUMBER(12,2),
+  min_threshold_date                DATE,
+  max_Threshold_date                DATE,
+  threshhold_date                   DATE
+);
+
+*/
+
+/* This table could exist in some form in both databases */
+CREATE TABLE journeys (
+  journey_id                        INTEGER PRIMARY KEY,

```

Comment:
Probably need to add a `UUID` type column to hold the reference to the Journey in the graph

Reply:
Have changed from INTEGER to UUID